CHANGES.rst

```diff
@@ -6,6 +6,7 @@ Version 0.8.2 (UNRELEASED)
 
 - Adds support for Python 3.10.
 - Adds workflow name validation utility.
+- Fixes file system usage calculation in ``get_disk_usage`` utility function.
```

Comment:
Perhaps add "...calculation on CephFS shares..." because the problem concerned CehpFS shares only.

Reply:
I will add. Although technically, it fixes the `du` command, and whether it is CephFS or not doesn't matter in the code.